### ISSUE TITLE
Semi-transparent images in Image-Outline mode obscure base imagery

### ISSUE THREAD

**Issue by mathewlippincott** [publiclab/mapknitter#225]
The Image Outline mode no longer displays just the outline, but also a semi-transparent image, obscuring base imagery and making it difficult to align another image to the base imagery if an image is already underneath. I'm not sure if this was a feature change from Mapknitter 1 to 2 or a bug that has crept in, as there is no longer much differentiation between Image Outline and Image Transparency modes. 

![screen shot 2016-09-12 at 1 05 17 pm](https://cloud.githubusercontent.com/assets/1203280/18451005/288a3c16-78ea-11e6-94c3-a9af00f393e9.png)

**Comment by gauravano** [comment]
Hi @mathewlippincott, first of all sorry for such a looong delay in responding and thanks for writing about the issue faced. @jywarren what do you think about this issue? 

Thanks!

**Comment by jywarren** [comment]
Yes I think it's an issue because we no longer render the images in a
canvas. I tried a number of fixes using css image outlines and think
there's still some promise there, although the edge gets distorted too. The
other alternative that might be cleaner is to use a native leaflet polygon
and maintain it aligned with the image edges manually. Thanks Gaurav and
Mathew.
